### PR TITLE
Added Device

### PR DESCRIPTION
**Pull request by SkynetHome**
Added Z-Wave Smart Switch ZW4005 to database and product file.

### CODE REVIEW

**Comment by SkynetHome** [comment]
I think I finally got this right! I hope you havent given up on me yet Chris!

File: bundles/binding/org.openhab.binding.zwave/database/ge/SmartSwitch.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (watou) [inline bundles/binding/org.openhab.binding.zwave/database/ge/SmartSwitch.xml]: Minor: I noticed that the database XML file names all seem to be lowercase, like smartswitch.xml.  SmartSwitch.xml might be the only CamelCase file name in the database?

**Comment by SkynetHome** [comment]
Lol...figures that I would still manage to screw something up! I would be happy to change that if I can get this successfully merged. I have been through hell and high water trying to learn how to do this!

**Comment by watou** [comment]
I *really* don't want to discourage -- your contributions are much appreciated by all Z-Wave binding users.  Try this:
```
cd bundles/binding/org.openhab.binding.zwave/database/ge
git mv SmartSwitch.xml smartswitch.xml
cd ..
edit products.xml to rename it in that file, too
git push origin master
```
That should update your pull request.  In future, after making your fork current, do a
```
git checkout -b my-database-update-or-some-other-branch-name
```
and use your private branch for the pull request.

**Comment by watou** [comment]
Updated my example for how to rename the file.  Sorry for incomplete advice earlier! :camel:

**Comment by SkynetHome** [comment]
No worries I am just about beyond dissuading! I created this using xoxos eclipse IDE. I have had hell with unmergable pulls. And have deleted recloned reinstalled and recreated workspaces countless times in the last two days. Is the file name something that is just an eye sore or is it going to cause problems. I have another binding update ready to commit and this time I was smart enough to add a branch. I would have already comitted it but I am deathly afraid of the un-mergable condition as I still do not understand what causes it and how to prevent it. Can I safely edit the file in in the IDE thus not going through the command line method above. I have had to reacquaint myself with the command terminal since getting into raspberry pi 2 which makes me feel like I am 11 years old all over again tampering with the emm386 emulator in DOS...That is to say I would much prefer to avoid the console as much as possible. If that is just part of working on this project then I will get use to it!

**Comment by SkynetHome** [comment]
Also I am comitting through the Github Windows app so I am created my branches there.

**Comment by SkynetHome** [comment]
Done new compare pull submitted

**Comment by watou** [comment]
One good alternative to command line use of git is SourceTree from Atlassian.  I don't use it frequently, but it's very capable and I'm sure there is an equivalent to "git mv" in it.  It will save you from having to cd (change directory) and remember a number of manual git commands.  Full speed ahead!

**Comment by SkynetHome** [comment]
Just downloaded that yesterday on the advice of Chris Jackson still
figuring it out.  Thanks for  the advice!

**Pull request closed without merging**